debug/bright-v7.json

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
Load this from https://www.mapbox.com/mapbox-gl-styles/styles/bright-v7.json rather than copying the style into the repository.